exampleMultipartyVideo/src/main/java/com/twilio/examplemultipartyvideo/MultiPartyActivity.java

```diff
@@ -587,6 +594,23 @@ public void onDisconnected(Room room, TwilioException e) {
                 }
             }
 
+            @Override
+            public void onDominantSpeakerChanged(@NonNull Room room, @Nullable RemoteParticipant remoteParticipant) {
+                if (dominantSpeakerImg.getParent() != null) {
```

Comment:
I think we should define a simple custom view that contains the video texture view and the image view. `ParticipantView`. You could then just toggle the visibility of the dominant speaker icon rather than adding and removing the image view.